library/consul

```diff
@@ -1,5 +1,11 @@
 Maintainers: Consul Team <consul@hashicorp.com> (@hashicorp/consul)
 
+Tags: 1.7.0-beta2, beta
+Architectures: amd64, arm32v6, arm64v8, i386
+GitRepo: https://github.com/hashicorp/docker-consul.git
+GitCommit: 8050b460ae441ba70072f1bf0c50c2cc456338eb
```

Comment:
The CI is failing because this commit either needs to be on the `master` branch or you need to add `GitFetch: refs/heads/beta` here. :+1:

Reply:
Thanks, I went ahead and merged it into the master branch.